internal/credential/vault/repository_credentials.go

```diff
@@ -38,6 +40,12 @@ func (r *Repository) Issue(ctx context.Context, sessionId string, requests []cre
 		if err != nil {
 			return nil, err
 		}
+
+		runJobsInterval := r.scheduler.GetRunJobsInterval()
+		if cred.getExpiration() < runJobsInterval {
+			event.WriteError(ctx, op, fmt.Errorf("WARNING: credential expires before job scheduler can run"), event.WithInfo("runJobsInterval", runJobsInterval))
```

Comment:
Sorry :)
So I am only catching this now, but I noticed this is in a loop and if multiple credentials have expiration lower than the scheduler run interval we will spam the same message. I think multiple events are fine, but perhaps we should add the credential library public id in the `withInfo`

Reply:
We might also not want this to be an error, perhaps `WriteSysEvent` or `WriteObservation` I would scan the code/look at the eventer documentation and validate which is better for a warning